Automattic/woocommerce-payments
# Hide upload buttons, amend placeholders in submitted evidence form

Fixes #495 

### Changes proposed in this Pull Request

When dispute details are submitted in the evidence form : 
* Hide upload buttons
* Show file name, if there was an evidence file uploaded
* If evidence file was not uploaded show a placeholder text - `Evidence file was not uploaded`
* Hide helper / placeholder texts in each section and field
* Show a notice mentioning details are read-only on the submitted form

### Screenshots

<details><summary>Before</summary>

![image](https://user-images.githubusercontent.com/4162931/194595603-42a4b10d-5013-49bb-bf26-e695938df85b.png)

</details>

<details><summary>After</summary>

![image](https://user-images.githubusercontent.com/4162931/194596024-f5784542-2dbd-46ac-9c9e-8422ce601b5e.png)

![image](https://user-images.githubusercontent.com/4162931/194596276-495fa007-63a7-44f7-ab0d-6eee95bcd35f.png)

</details>



#### Testing instructions

1. Do a test purchase and create a dispute transaction using[ Stripe test card for dispute](https://stripe.com/docs/testing#disputes) case - `4000 0000 0000 1976` 
2. Browse to `WP Admin > Payments > Disputes and click on the dispute that appears there
3. Click on `Challenge Dispute` button
4. On the page that appears next, select a Product Type in the dropdown. 
5. Make sure all fields appear correctly in the resulting form, with placeholder texts wherever applicable. Fields should be editable at this stage
6. After this step, try out three scenarios: 

##### a. Submit with all fields filled
i. Submit the evidence with all fields filled in the form. Make sure you also upload any small test file for each of the uploads. 
ii. After submission, browse to `Payments > Disputes` and select `All disputes` in dropdown. Select the recently submitted evidence. Click `View submitted evidence` button.
iv. You should  see a notice on top indicating the fields are all read only, similar to the screenshot: 
![image](https://user-images.githubusercontent.com/4162931/194605000-4481a54b-ee03-413e-919a-2784753488d2.png)
v. Text fields and date should show what you entered during submission, and it should not be editable
vi. There should be no upload buttons visible for files, instead, you should see file names of uploaded evidence for each field
vii. There should be no placeholder / help text indicating what a field / section is meant for. ( no longer helpful since fields are already submitted ) 

##### a. Submit with no fields filled

i. Do not fill up any field, other than those prepopulated ( User name , address etc ) . Do not upload any proof documents either.
ii. Submit the evidence, browse to `Payments > Disputes` and select `All disputes` in dropdown. Select the recently submitted evidence. Click `View submitted evidence` button.
iii. Text fields where there was no information submitted should show text - `No information submitted`
iv. File upload sections should not show any upload buttons and should show - `Evidence file was not uploaded`

-------------------


**Post merge**
